CMakeLists.txt

```diff
@@ -81,6 +90,15 @@ if(LVK_DEPLOY_DEPS)
   lvk_set_folder(LVKDependencies "LVK")
 endif()
 
+if(LVK_WITH_WAYLAND)
+  include(${LVK_ROOT_DIR}/third-party/deps/src/cmake-wayland/FindWayland.cmake)
+  if(WAYLAND_FOUND)
+    add_definitions(-DLVK_WITH_WAYLAND=1)
```

Comment:
It should be exported with `target_compile_defenitions`. Otherwise, LVK users will not see this definition.